build_id_getter.go

```diff
@@ -21,16 +21,16 @@ func NewBuildIDGetter(finishedStages model.FinishedStages, targetNames []string)
 }
 
 func (bg BuildIDGetter) GetBuildIDs() ([]string, error) {
-	var buildIDs []string
```

Comment:
I think you could cover this with the tests easily just by adding a duplicate build id into the input data